src/App.tsx

```diff
@@ -41,6 +43,7 @@ export const App = () => {
       <ModalProvider />
       <Toaster />
       <CreateStrategyCTAMobile />
+      {!isNativeApp && <Footer />}
```

Comment:
It's good to put the footer at the root to make sure its landmark is maintain.
I would just have put it just after the main tag, this is the logical HTML flow.